packages/select-rich/src/LionSelectRich.js

```diff
@@ -206,6 +206,13 @@ export class LionSelectRich extends ScopedElementsMixin(
     this.__initInteractionStates();
   }
 
+  connectedCallback() {
+    this._listboxNode.registrationTarget = this;
```

Comment:
putting this before super.firstUpdated is too late, because there, the firstUpdated of the listboxNode is already called. If we put it before super.connectedCallback, we're on time, because the firstUpdated of the listboxNode hasn't been called yet, because no render has happened yet.